### ISSUE TITLE
Error on 'interesting' remote setup

### ISSUE THREAD

**Issue by jimhester** [r-lib/usethis#1173]
My remotes are set like this

```
> git remote -v
origin  git@github.com:jimhester/usethis.git (fetch)
origin  git@github.com:jimhester/usethis.git (push)
upstream        git@github.com:r-pkgs/usethis.git (fetch)
upstream        git@github.com:r-pkgs/usethis.git (push)
```

And when running `usethis::git_sitrep()` I get the following

``` r
setwd("~/p/usethis")
library(usethis)
git_sitrep()
#> Git config (global)
#> * Name: 'Jim Hester'
#> * Email: 'james.f.hester@gmail.com'
#> * Vaccinated: TRUE
#> usethis + gert
#> * Default usethis protocol: <unset>
#> * gert supports HTTPS: TRUE
#> * gert supports SSH: TRUE
#> GitHub
#> * Personal access token: '<found in env var>'
#> * User: 'jimhester'
#> * Name: 'Jim Hester'
#> * Email(s): '<unknown>'
#> Repo
#> * Path: '/Users/jhester/projects/usethis'
#> * Local branch -> remote tracking branch: 'master' -> 'upstream/master'
#> GitHub pull request readiness
#> Error in utils::modifyList(cfg, type = "fork_upstream_is_not_origin_parent", : unused arguments (type = "fork_upstream_is_not_origin_parent", unsupported = TRUE, desc = glue("\n      The {ui_value('origin')} GitHub remote is a fork, but it's not a fork \\\n      of the repo configured as the {ui_value('upstream')} remote.\n      "))
```

<sup>Created on 2020-07-06 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by jennybc** [comment]
This seems weird (and is what we're reporting but the reporting function seems to be broken for unsupported configs (same as #1175):

```
upstream        git@github.com:r-pkgs/usethis.git (fetch)
```

Did you mean to be pointing at `r-pkgs` instead of `r-lib`?

**Comment by jimhester** [comment]
yeah, `r-pkgs` was the old name of the org, and I just haven't bothered to change it.

**Comment by jennybc** [comment]
> yeah, r-pkgs was the old name of the org

I did not know that! 🤯

I think I'm comfortable calling this setup out as weird, though. I don't know how I could really follow the redirect and it doesn't seem extreme to nudge people to update. It shouldn't happen to people very often.

So I'm going to close as a duplicate of #1175.

**Comment by jennybc** [comment]
While working on #1175, I looked into this a bit more.

There is no evidence in the regular payload when you GET a repo that it once went by a different name, i.e. different owner and/or different repo name.

So I still don't plan to do anything about this.

But I did learn there's an undocumented `/repositories/:id` endpoint, that one could use to figure out that two URLs that look different actually point at the same thing.
